smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpBindingProtocolGenerator.kt

```diff
@@ -91,9 +94,8 @@ fun formatHeaderOrQueryValue(
     memberShape: MemberShape,
     location: HttpBinding.Location,
     bindingIndex: HttpBindingIndex,
-    defaultTimestampFormat: TimestampFormatTrait.Format
+    defaultTimestampFormat: TimestampFormatTrait.Format,
```

Comment:
nit: Why are there `,` added after last argument of function calls? (Not just here, but a dozen other places as well)

Reply:
I installed ktlint locally which started doing that every time I did a lint on the file, will check that kotlin style guide suggests